homework.js

```diff
@@ -0,0 +1,113 @@
+// 1.Create a function that will get the sum of the numbers between 1 and n and return the answer
+
+function summation(n)
+ {
+  let sum = 0;
+  for (let i = 1; i <= n; i++) 
+      {
+    sum += i;
+      }
+  return sum;
+}
+
+console.log( summation(5) );
+
+
+// 2. Create a program to get the sum of all the even numbers in a group 
+
+function summationEven(n)
+ {
+  let sum = 0;
+  for (let i = 1; i <= n; i++) 
+      {
+if(i % 2 == 0)
+    sum += i;
+      }
+  return sum;
+}
+
+console.log( summationEven(5) );
+
+// 3.Create a function to get the average of a group of numbers 
+
+avrg = [2, 2, 8, 4, ];
+
+for ( var i = 0; i < avrg.length; i ++){
+    var avg = (avrg[i] / avrg.length) * avrg.length
```

Comment:
In order to get the average, you need to first sum all the elements of your array then divide by the array length which is the count of your elements.